quadrat/experimental-theme.json

```diff
@@ -167,8 +167,8 @@
 			},
 			"post-content": {
 				"padding": {
-					"left": "var(--wp--custom--margin--horizontal)",
-					"right": "var(--wp--custom--margin--horizontal)"
+					"left": "5vw",
+					"right": "5vw"
```

Comment:
> why do we need it to be a responsive value if we only actually see the effect when we are on mobile?

because this is the same spacing we use in the header